frontend/src/metabase/nav/containers/Navbar.jsx

```diff
@@ -354,21 +369,36 @@ export default class Navbar extends Component {
               <h4 className="hide sm-show ml1 text-nowrap">{t`Ask a question`}</h4>
             </Link>
           )}
+
           {hasDataAccess && (
             <Link
               mr={[1, 2]}
-              to="browse"
+              to={this.props.plainNativeQuery.question().getUrl()}
               p={1}
-              className="flex align-center rounded transition-background"
-              data-metabase-event={`NavBar;Data Browse`}
               hover={{
                 backgroundColor: darken(color("brand")),
               }}
+              className="flex align-center rounded transition-background"
+              data-metabase-event={`NavBar;SQL Question`}
             >
-              <Icon name="table_spaced" size={14} />
-              <h4 className="hide md-show ml1 text-nowrap">{t`Browse Data`}</h4>
+              <Icon name="sql" size={18} />
+              <h4 className="hide sm-show ml1 text-nowrap">{t`SQL Question`}</h4>
             </Link>
           )}
+          {hasNativeWrite && (
+            <IconWrapper
+              className="relative hide sm-show mr1 overflow-hidden"
+              hover={NavHover}
+            >
+              <Link
+                to="browse"
+                className="flex align-center"
+                data-metabase-event={`NavBar;Browse Data`}
+              >
+                <Icon size={18} p={"11px"} name="reference" tooltip={t`Browse Data`} />
```

Comment:
could we use a different icon since this is already used to mean something else (data catalog) in the SQL editor?

https://www.flaticon.com/free-icon/analytics_622666?term=search%20data&page=1&position=2